src/main/java/com/stripe/model/Card.java

```diff
@@ -34,6 +34,12 @@
 	String status;
 	String tokenizationMethod;
 
+	// Please note that these field are for internal use only and are not typically returned
+	// as part of standard API requests.
+	String description;
```

Comment:
@zackstripe I think it's due to the way we built the lib. See brandur's comment here: https://github.com/stripe/stripe-java/pull/309#discussion_r77237022